### ISSUE TITLE
训练nextvlad模型，Hit@1不会变化

### ISSUE THREAD

**Issue by lale314** [PaddlePaddle/models#4814]
用models/PaddleCV/video/里的代码和youtube8m数据集训练nextvlad模型，为啥Hit@1上不去？

[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:54:35] Epoch 10, iter 41910, time 0.24196863174438477,  , loss = 2.364186, Hit@1 = 0.08, PERR = 0.88, GAP = 0.93
[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:54:37] Epoch 10, iter 41920, time 0.23306751251220703,  , loss = 2.527951, Hit@1 = 0.05, PERR = 0.91, GAP = 0.93
[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:54:45] Epoch 10, iter 41930, time 0.25219297409057617,  , loss = 2.148221, Hit@1 = 0.05, PERR = 0.89, GAP = 0.93
[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:54:54] Epoch 10, iter 41940, time 0.23409438133239746,  , loss = 2.436858, Hit@1 = 0.05, PERR = 0.90, GAP = 0.93
[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:55:02] Epoch 10, iter 41950, time 0.2435312271118164,  , loss = 2.446642, Hit@1 = 0.06, PERR = 0.88, GAP = 0.93
[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:55:11] Epoch 10, iter 41960, time 6.53504204750061,  , loss = 1.978944, Hit@1 = 0.05, PERR = 0.90, GAP = 0.95
[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:55:14] Epoch 10, iter 41970, time 0.2472538948059082,  , loss = 2.467091, Hit@1 = 0.05, PERR = 0.93, GAP = 0.93
[INFO: metrics_util.py:   79]: [TRAIN 2020-08-26 07:55:24] Epoch 10, iter 41980, time 0.24446415901184082,  , loss = 2.176183, Hit@1 = 0.04, PERR = 0.90, GAP = 0.94

**Comment by huangjun12** [comment]
请修改PaddleCV/video/metrics/youtube8m/eval_util.py第40行
https://github.com/PaddlePaddle/models/blob/b87761f8100a545e0015046dd55d886ce90c190e/PaddleCV/video/metrics/youtube8m/eval_util.py#L40
为
hits = actuals[numpy.arange(actuals.shape[0]), top_prediction]

**Comment by huangjun12** [comment]
修复Pr: https://github.com/PaddlePaddle/models/pull/4918
